src/Kdyby/Monolog/Diagnostics/MonologAdapter.php

```diff
@@ -41,7 +41,7 @@ class MonologAdapter extends Logger
 	/**
 	 * @var Monolog\Logger
 	 */
-	private $monolog;
+	protected $monolog;
```

Comment:
this is not needed, since you can catch the constuctor argument in the inherited class